### PR TITLE
Add version file, update build parameters and other fixes

### PR DESCRIPTION
**Pull request by khansaad**
Signed-off-by: Saad Khan <saakhan@redhat.com>

### CODE REVIEW

**Comment by dinogun** [comment]
@khansaad Pls fix the conflicts, thanks

File: deploy_hpo.sh [hunk 1]
```diff
@@ -17,23 +17,27 @@
 
 ROOT_DIR="${PWD}"
 SCRIPTS_DIR="${ROOT_DIR}/scripts"
-HPO_REPO="kruize/hpo"
-HPO_VERSION="0.0.1"
+HPO_REPO="saakhan/hpo"
```
Comment (dinogun) [inline deploy_hpo.sh:20]: This needs to be `kruize/hpo`

Comment (dinogun) [inline deploy_hpo.sh:20]: This needs to be `kruize/hpo`

File: src/hpo_service.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import threading
 import json
 from bayes_optuna import optuna_hpo
-from typing import TypedDict
+#from typing import TypedDict
```
Comment (dinogun) [inline src/hpo_service.py:4]: Please remove this code, we should not be merging commented code

Comment (dinogun) [inline src/hpo_service.py:4]: Please remove this code, we should not be merging commented code

File: src/service.py [hunk 3]
```diff
@@ -137,6 +130,14 @@ def get_search_create_study(search_space_json, operation):
     if operation == "EXP_TRIAL_GENERATE_NEW":
         experiment_name, total_trials, parallel_trials, direction, hpo_algo_impl, id_, objective_function, tunables, value_type = get_all_tunables(
             search_space_json)
+        if(not total_trials):
+            return -1
+        if(not parallel_trials):
+            parallel_trials = n_jobs
```
Comment (dinogun) [inline src/service.py:136]: 1. Log an error message here if the total_trials is not set
2. parallel trials cannot be anything other than "1", log an error message for that as well
3. Space after if. ` if ()`

Comment (dinogun) [inline src/service.py:136]: 1. Log an error message here if the total_trials is not set
2. parallel trials cannot be anything other than "1", log an error message for that as well
3. Space after if. ` if ()`

Comment (khansaad) [reply src/service.py]: 1. Added a required condition in json_validate for total_trials
2. Added exception for parallel trials value other than 1

File: src/hpo_service.py (hunk 2 above)
Comment (khansaad) [reply src/hpo_service.py]: updated

File: deploy_hpo.sh (hunk 1 above)
Comment (khansaad) [reply deploy_hpo.sh]: updated

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**